test/TorchSharpTest/TestTorchAudioModels.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using Xunit;
+
+using TorchSharp;
+
+namespace TorchSharp
+{
+    public class TestTorchAudioModels
+    {
+        [Fact]
+        public void Tacotron2Model()
+        {
+            var tacotron2 = torchaudio.models.Tacotron2(n_symbol: 96);
```

Comment:
For such a large edit, with so many parameters, I would like to see a few more validations here.

Reply:
Added more unit tests
https://github.com/kaiidams/TorchSharp/commit/c798130c41f860758116e5842dadd4cb7daedf89